client/app/login/login.html

```diff
@@ -0,0 +1,44 @@
+<div class="container">
+	<div class="row">
```

Comment:
good catch, i snagged this code from the interwebs. i will clean it up a bit and push a little later on when i have some free time